keel-orca/src/main/kotlin/com/netflix/spinnaker/keel/model/EchoNotification.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ *
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.keel.model
+
+import com.netflix.spinnaker.keel.api.NotificationConfig
+import com.netflix.spinnaker.keel.api.NotificationFrequency
+import com.netflix.spinnaker.keel.api.NotificationFrequency.NORMAL
+import com.netflix.spinnaker.keel.api.NotificationFrequency.QUIET
+import com.netflix.spinnaker.keel.api.NotificationFrequency.VERBOSE
+import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_COMPLETE
+import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_FAILED
+import com.netflix.spinnaker.keel.model.NotificationEvent.ORCHESTRATION_STARTING
+
+data class EchoNotification(
+  val type: String,
+  val address: String,
+  val `when`: List<String>,
+  val level: String = "pipeline",
+  val message: Map<String, NotificationMessage>
+)
+
+enum class NotificationEvent {
+  ORCHESTRATION_STARTING {
+    override fun text(): String = "orchestration.starting"
+    override fun notificationMessage(): NotificationMessage = NotificationMessage("$RAINBOW Managed update starting")
```

Comment:
I definitely think it would make sense to provide context around _what_ is being updated. Ideally a link too. A standard method on `ResourceSpec` to give a human-readable description of what it is?

Reply:
This will already provide a link to the orchestration(task) in the ui - same way as normal pipeline notification works. FYI this is the tiny notification that pops up in a slack message.

<img width="833" alt="Screen Shot 2019-10-03 at 1 45 09 PM" src="https://user-images.githubusercontent.com/8454927/66162896-0d4d8880-e5e4-11e9-8b68-c8f9c74942b6.png">

Reply:
That's basically what it looks like, and the task description is a clickable link to the task in the UI. Given that information, what more should we include? (this is easy to update in the future)

Reply:
Looks good to me